care/facility/tests/test_external_result_api.py

```diff
@@ -12,6 +13,69 @@ def setUpTestData(cls) -> None:
         cls.local_body = cls.create_local_body(cls.district)
         cls.ward = cls.create_ward(cls.local_body)
         cls.user = cls.create_super_user("su", cls.district)
+        cls.external_result = PatientExternalTest.objects.create(
+            district=cls.district,
+            srf_id="00/EKM/0000",
+            name="Test Upload0",
+            age=24,
+            age_in="years",
+            gender="m",
+            mobile_number=8888888888,
+            address="Upload test address",
+            ward=cls.ward,
+            local_body=cls.local_body,
+            source="Secondary contact aparna",
+            sample_collection_date="2020-10-14",
+            result_date="2020-10-14",
+            test_type="Antigen",
+            lab_name="Karothukuzhi Laboratory",
+            sample_type="Ag-SD_Biosensor_Standard_Q_COVID-19_Ag_detection_kit",
+            patient_status="Asymptomatic",
+            is_repeat=True,
+            patient_category="Cat 17: All individuals who wish to get themselves tested",
+            result="Negative",
+        )
+
+    def test_list_external_result(self):
+        response = self.client.get("/api/v1/external_result/")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_retrieve_external_result(self):
+        response = self.client.get(
+            f"/api/v1/external_result/{self.external_result.id}/"
+        )
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
assert the response as well

Reply:
what should I test in `test_list_external_result` , i added a test for checking count, what else?

Reply:
you can add more external results objects and use filters defined [here](https://github.com/coronasafe/care/blob/db6f739a32310a6cc80e36d8c8395fdf01937aa0/care/facility/api/viewsets/patient_external_test.py#L53) to see if the filters are returning expected objects